mysql-test/suite/galera/t/pxc_install_gr.test

```diff
@@ -0,0 +1,50 @@
+#
+# Test Galera as a slave to a MySQL master
+#
+# The galera/galera_2node_slave.cnf describes the setup of the nodes
+#
+#
+
+--source include/galera_cluster.inc
+
+--connection node_2
+
+--disable_query_log
+call mtr.add_suppression("Plugin group_replication reported");
+call mtr.add_suppression("Group replication cannot be used with Percona XtraDB Cluster in strict mode.");
+call mtr.add_suppression("Plugin 'group_replication' init function returned error.");
+call mtr.add_suppression("Using group replication with Percona XtraDB Cluster is only supported for migration");
+--enable_query_log
+
+--connection node_1
+
+--disable_query_log
+call mtr.add_suppression("Plugin group_replication reported");
+call mtr.add_suppression("Group replication cannot be used with Percona XtraDB Cluster in strict mode.");
+call mtr.add_suppression("Plugin 'group_replication' init function returned error.");
+call mtr.add_suppression("Using group replication with Percona XtraDB Cluster is only supported for migration");
+--enable_query_log
+
+# Default
+SET GLOBAL pxc_strict_mode = 0;
+
+--sleep 2
```

Comment:
Why do we need these sleeps?

Reply:
We only really need the last one, and even that's a rare failure: if we change node_2's strict mode back to 0 before it executes the install plugin command, it will succeed, causing check_mysqld to fail. But this is just an instability, as next time the test will most likely pass with retry-pass.

Reply:
I agree that this is not critical, as code fix is OK, but anyway potential source of MTR instability which, well..., will cause constant pain and probably will never be fixed as "it will fail only sometimes".
Maybe this can help: https://www.percona.com/doc/percona-xtradb-cluster/LATEST/wsrep-system-index.html#wsrep_sync_wait

Or at least it seems to be ease to synchronize using DEBUG_SYNC...